src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -130,6 +127,7 @@ public static void assertCommandSuccess(Command command, Model actualModel, Comm
             Model expectedModel) {
         try {
             CommandResult result = command.execute(actualModel);
+            System.out.println(result.getFeedbackToUser() + " | " + expectedCommandResult.getFeedbackToUser());
```

Comment:
Could be removed, unless the line printed is there for a purpose other than testing

Reply:
Sure